tests/integration/cairo1/misc/test_panic.py

```diff
@@ -0,0 +1,29 @@
+from pathlib import Path
+
+import pytest
+
+from tests.integration._conftest import ProtostarProjectFixture
+from tests.integration.conftest import (
+    CreateProtostarProjectFixture,
+    assert_cairo_test_cases,
+)
+
+
+@pytest.fixture(name="protostar_project", scope="function")
+def protostar_fixture(create_protostar_project: CreateProtostarProjectFixture):
+    with create_protostar_project() as protostar_project:
+        yield protostar_project
+
+
+async def test_panic(protostar_project: ProtostarProjectFixture, datadir: Path):
+    testing_summary = await protostar_project.protostar.test(
+        datadir / "panic_test.cairo",
+    )
+
+    assert_cairo_test_cases(
+        testing_summary,
+        expected_failed_test_cases_names=[
+            "test_panic_when_felt252_not_short_str",
+            "test_panic_with_felt252_max_val",
+        ],
+    )
```

Comment:
I think we should check the output just like in `tests/integration/cairo1/compiler_bindings/test_print.py`

Reply:
Added